auto_configure.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+echo 'Please _please_ make sure you have adjusted the SD volume location & the disk ID'
+echo 'Sleeping for 5s for safety'
+sleep 5
+
+touch /Volumes/boot/ssh
+cp ./config.txt /Volumes/boot/config.txt
```

Comment:
config.txt can change, but the two lines probably won't. Maybe `tee` could be used to top the new lines into the end of the existing file?